proxy/src/graphql/project.rs

```diff
@@ -63,12 +63,39 @@ impl From<meta::Project> for Metadata {
     }
 }
 
+/// Shows if a project exists on the Registry and distinguishes between Org and User owned.
+#[derive(GraphQLEnum)]
+pub enum Registered {
+    /// Project is not present on the Registry.
+    Not,
+    /// Project is registered under an Org.
+    Org,
+    /// Project is registered under a User.
+    User,
+}
```

Comment:
I'm thinking this might be annoying to use when you just want to know registered or not. Do we need to differentiate whether it's registered under an org or a user somewhere? Alternatively you could implement a `impl From<Registered> for bool`, so at least you can do `if registered.into() { .. }`, but I wonder if the distinction is necessary in the first place :thinking:

Reply:
Was planning on extending this to variants which carry the respective **org id** or **user handle**. An `Into` for `bool` sounds like a fun idea.

Reply:
Ah in that case it makes sense, if you are planning to use a different type for orgs vs user ids, since underneath they have the same format. I'll leave it up to you.

Reply:
Dropped this enum for now and we revise later.

Reply:
I looked at the designs again (and had a chat with @juliendonck), we're using this only for display a "registered" badge in the UI. I also don't see why we would ever want to differentiate whether it's under a project or user, so a bool would be much easier to use in the front-end.

Reply:
Addressed.